helm_deploy/prometheus-custom-rules-resettlement-passport.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  namespace: hmpps-resettlement-passport-dev
```

Comment:
Don't include namespace as this will be set by the circleci build

Reply:
Done